src/elements/02-elements/shortcut/shortcut.hbs

```diff
@@ -0,0 +1,8 @@
+<a href="#" class="ys-shortcut" title="Music">
+    <figure class="ys-shortcut__icon-container">
```

Comment:
```suggestion
    <figure class="ys-shortcut__icon-container" aria-hidden="true">
```

I'm suggesting adding `aria-hidden` to the icon part of the shortcut just like we do with buttons etc.